File: utn-labcomII-practices/Unidad-4/practica-1/ej-6-p1.js
// Escribe un programa que pida dos números y escriba en la pantalla cual es el mayor.

let numero = parseFloat(prompt("Ingrese un número: "));
let numero2 = parseFloat(prompt("Ingrese otro número: "));

if (numero > numero2) {
    document.write("El mayor número es: " + numero);
} else if (numero < numero2) {
    document.write("El mayor número es: " + numero2);
} else {
    document.write("Ambos números son iguales.");
}

File: utn-labcomII-practices/Unidad-5/practica-1/ej-6-p1.py
#Escribe un programa que pida 3 números y escriba en la pantalla el mayor de los tres.

numero = float(input("Ingrese un número: "))
numeroDos = float(input("Ingrese otro número: "))
numeroTres =  float(input("Ingrese otro número: "))

if(numero>numeroDos and numero>numeroTres):
    print("El mayor número es: ", numero)
elif(numeroDos>numeroTres):
    print("El mayor número es: ", numeroDos)
else:
    print("El mayor número es: ", numeroTres)